### DISCUSSION TITLE
Navigation simulation

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#73] [Q&A] [answered]
How to show the next button in v_3.0.12 ?
![Cruiser_navigation_simulation](https://user-images.githubusercontent.com/54167201/223953159-b0e1fe56-ef6c-47ea-8525-d86696076028.png)

**Comment by devemux86** [comment]
Navigation simulation is simpler now with only one press.

- It was already possible to start simulation anywhere on the route: _move the map near any turn instruction_.
- And press the navigation button to start / stop the simulation (without filling the screen with more buttons).

**Comment by 0709wiwiwi** [comment]
And at or near (complex) (out and return) track overlaps, the selection accuracy by the buttons was unbeatable simple.

**Reply by devemux86** [reply]
We can start a turn instruction or two earlier in overlapping areas, so it can snap into the correct path.

**Comment by devemux86** [comment]
Next Cruiser will have new options in navigation:
- **Normal**: _real time_ navigation using GPS location
- **Simulation**: run the navigation in _simulation_ at adjustable speed
- **Static**: quickly explore the route and listen the voice instructions

There will also be a new user interface in simulation / static navigation modes.

<img src="https://user-images.githubusercontent.com/3484020/230918786-2743dc8c-4f82-4385-a83a-e131e3a8ed02.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/231140327-15cfb5d5-0048-4a6c-98b1-6d870f2ba46d.jpg" width="33%"/><img src="https://user-images.githubusercontent.com/3484020/231140351-b0c91742-5bab-4ab7-86f1-fd9b86c0c578.jpg" width="33%"/>

**Comment by devemux86** [comment, marked as answer]
New navigation simulation modes are available in [Cruiser 3.0.18](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

For more details please see above:
- https://github.com/devemux86/cruiser/discussions/73#discussioncomment-5571203

**Reply by DolomitesTom** [reply]
works

**Reply by devemux86** [reply]
Navigation simulation modes were already in the Cruiser code, so they are just now publicly available to everyone. :slightly_smiling_face:
